File: leetcode/10-99/58. Length of Last Word/solution.cs
public class Solution
{
    public int LengthOfLastWord(string s)
    {
        string word = "";

        for (int i = s.Length - 1; i >= 0; --i)
        {
            if (word.Length == 0 && s[i] == ' ')
                continue;
            else
            {
                if (s[i] == ' ')
                    return word.Length;

                word += s[i];
            }
        }

        return word.Length;
    }
}

File: leetcode/10-99/58. Length of Last Word/solution.java
class Solution 
{
    public int lengthOfLastWord(String s) 
    {
        String word = "";
        
        for (int i = s.length() - 1; i >= 0; --i)
        {
            if (word.isEmpty() && s.charAt(i) == ' ')
                continue;
            else
            {
                if (s.charAt(i) == ' ')
                    return word.length();
                
                word += s.charAt(i);
            }
        }
        
        return word.length();
    }
}